tools/git_lfs.rst

```diff
@@ -52,8 +54,6 @@ First, add these lines into your :file:`~/.gitconfig` file:
 
 .. literalinclude:: snippets/git_lfs_gitconfig.txt
```

Comment:
@jmatt Don't forget to edit `snippets/git_lfs_gitconfig.txt` to remove the `batch = False` part.